src/common/configs/visualization-configuration-factory.ts

```diff
@@ -56,6 +56,7 @@ export class VisualizationConfigurationFactory {
                     toggleLabel: null,
                     linkToDetailsViewText: null,
                 },
+                shouldShowExportReport: null,
```

Comment:
Do you have to add this?

Reply:
Yep, this returns a VisualizationConfiguration & not an AssessmentVisualizationConfiguration so it has to include it